manual/alshelp.css

```diff
@@ -0,0 +1,68 @@
+#body {
+overflow-y: scroll;
+overflow-x: hidden;
+}
+
+#left_top {
+position: absolute;
+top:0;
+left:0;
+height:8em;
+width:13em;
+background-color:ivory;
+}
+
+#left_bottom {
+position: absolute;;
+top:8em;
+left:0;
+height:100%;
+width:13em;
+background-color:cornsilk;
+}
+
+#right {
+position: absolute;;
+top:0;
+left:13em;
+height:120%;
```

Comment:
Looks like iframes don't interact well with position fixed/absolute. The work-around is to wrap them in divs:

http://stackoverflow.com/questions/4195971/iframe-with-a-css-fixed-position-possible